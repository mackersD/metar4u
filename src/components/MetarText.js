import React from 'react'

const MetarText = ({text}) => {
  return (
    <div>
      {text}
    </div>
  )
}

export default MetarText
