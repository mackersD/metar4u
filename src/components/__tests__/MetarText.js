import React from 'react'
import renderer from 'react-test-renderer'
import MetarText from '../MetarText'

it('MetarText - renders', () => {
  const tree = renderer.create(
    <MetarText />
  ).toJSON()
  expect(tree).toMatchSnapshot()
});

it('MetarText - expected rendering', () => {
  var text = "KAUS 091853Z 16008KT 10SM FEW065 FEW250 36/20 A3004 RMK AO2 SLP158 T03560200"
  var comp = renderer.create(
    <MetarText text={text} />
  ).toJSON()
  expect(comp).toMatchSnapshot()
})
