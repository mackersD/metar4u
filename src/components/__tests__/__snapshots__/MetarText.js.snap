// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MetarText - expected rendering 1`] = `
<div
  className="metarText"
>
  KAUS 091853Z 16008KT 10SM FEW065 FEW250 36/20 A3004 RMK AO2 SLP158 T03560200
</div>
`;

exports[`MetarText - renders 1`] = `
<div
  className="metarText"
/>
`;
